### PR TITLE
Implement popup

### PR DESCRIPTION
**Pull request by edenlisk**
In this branch I did the following:
1. Implemented popup for both mobile and desktop version

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -50,4 +50,393 @@ menuIcon.addEventListener('keydown', (e) => {
...
+          <span><img src="assets/popup/icons/see-live-icon.svg"></span>
+    `;
+
+  const seeSourceBtnDesktop = document.createElement('button');
+  seeSourceBtnDesktop.type = 'button';
+  seeSourceBtnDesktop.classList.add('see-project');
+  seeSourceBtnDesktop.innerHTML = `
+        See Source
+          <span><img src="assets/popup/icons/see-source-icon.svg"></span>
+    `;
```
Comment (Nemwel-Boniface) [inline script.js]: - [x] Great job making your popup buttons. Kindly also include the button links such that when a user clicks on the button for seeing live, it takes him / her to the respective projects live or GitHub repository.
- [x] Also, kindly reduce the padding in the buttons and increase the padding from the margins to the buttons in mobile view. Right now the design does not match the provided Figma design.

**This is how your Mobile version looks like**

image


**This is the desired design:**

image

Comment (Nemwel-Boniface) [inline script.js]: - [x] Good job working on your popup. Right now, when I click on any see project, the popup shows correctly. However, when you look keenly, you notice that somehow the popup is being centered but not at the page view port widths and heights but at the whole length of the webpage. This means that on mobile, I click on the popup then scroll down to see the popup. This is not a very effective way of having it. 

**To fix this,** Make the popup centered by the user's screen viewport height/ width and not the whole website's length.


**What I mean on mobile view:**

[Screencast from 17-11-2022 03:05:28 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/202441779-8748bceb-1f8c-4702-894e-0370bb7ab7ac.webm)

**Now what I mean on the desktop view**

[Screencast from 17-11-2022 03:06:24 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/202442082-5a263ff2-ec3b-4268-a472-a492069672f3.webm)


**Hope now** you understand what I mean and will address it.

- [x] Your design on mobile view has a much larger marins/ padding on the background. This then makes the other sections to get "squeezed" and distorts the overall design. Kindly make this change

**This is how your mobile version currently looks like:**

[Screencast from 17-11-2022 03:27:20 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/202446271-f001c9d1-9a57-4b2b-8a47-32ceba0c4c8c.webm)

**This is the desired design:**

image

File: script.js [hunk 2]
```diff
@@ -50,4 +50,393 @@ menuIcon.addEventListener('keydown', (e) => {
...
+
+  // popup mobile language item2
+  const popupLanguageMobileItem2 = document.createElement('li');
+  popupLanguageMobileItem2.innerHTML = [projectInfo[i].popupLanguageMobile[1]];
+  popupLanguageMobileItem2.classList.add('pro-lang');
+
+  // popup mobile language item3
+  const popupLanguageMobileItem3 = document.createElement('li');
+  popupLanguageMobileItem3.innerHTML = [projectInfo[i].popupLanguageMobile[2]];
+  popupLanguageMobileItem3.classList.add('pro-lang');
```
Comment (Nemwel-Boniface) [inline script.js:385]: - [x] Kindly check your project languages design. They are not aligned as it is in the Figma design. Kindly fix this.

**This is the desktop design:**

image

**This is the desired design:**

image

**Note** I have realized that the trend is the same for the mobile view as well. Kindly also make this design change to the mobile styling of the popup as well.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @edenlisk   :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] As this is a project that makes use of JavaScript, kindly ensure that you also set up the ESlint, the JavaScript linter. Follow [this link](https://github.com/microverseinc/linters-config/tree/master/javascript) to set it up.

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @edenlisk, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**